### ISSUE TITLE
{Linked:Bug2509191}[Accessibility][Visual Requirements -Workflow Notification Card]: Keyboard focus indicator is not properly visible on "view in portal" link in company name section.

### ISSUE THREAD

**Issue by palitsourish** [microsoft/AdaptiveCards#7717]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS Build: Windows 11
Version: Dev( Build 22526.1000)
Browser: Edge Dev
Version ( 98.0.1108.15) (Official build) (64-bit)
URL: [Designer | Adaptive Cards](https://adaptivecards.io/designer/)
Screen reader: Narrator

Repro Steps:
1. Open the above URL and login with valid credentials.
2. Workflow Notifications page will get displayed.
3. Now, navigate to 'View in portal' link in 'company name' section.
4. check whether the keyboard focus indicator is clearly visible or not.

Actual:
Keyboard focus indicator is not properly visible on "view in portal" link in company name section.
Focus indicator is visible only on two sides for the "View in portal" link.

Expected:
Keyboard focus indicator should be properly visible four sides on "view in portal" link in company name section.
Focus indicator(s) must be clearly visible on each interactive element when navigating the page using the tab key.

User Impact:
Keyboard only users need to know which element has focus so they can track their navigation through and interaction with interactive controls.

MAS Reference:
[MAS 2.4.7 - Focus Visible](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b3b98b04f-4fab-4d3e-81af-a0906a34849c%7d)

“Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fforms.office.com%2fpages%2fresponsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-lalapati%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656401002888328%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=9rKyqggThK5iHRPhG9iMiNp6zSq%2BYr%2fr4gmiXwwFsnM%3D&reserved=0) “

### Screenshots

<img width="958" alt="MAS2 4 7 Keyboard focus is not properly visible" src="https://user-images.githubusercontent.com/95914144/181213877-33555da8-6b2b-4d8f-9456-0e9b253458e8.png">

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is not repro hence closing this issue.
